.github/workflows/release.yml

```diff
@@ -1,19 +1,33 @@
-name: Release non-snapshot package to the Maven Central Repository
+name: Release New Version
 on:
-  release:
-    types: [created]
+  workflow_dispatch:
+    inputs:
+      releaseversion:
+        description: 'Release version'
+        required: true
 jobs:
   publish:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
     steps:
-      - name: Configure GPG Key
-        run: echo -n "$GPG_SIGNING_KEY" | base64 --decode | gpg --import --no-tty --batch --yes
-        env:
-          GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
-
       - uses: actions/checkout@v4
-
-      - name: Set up Maven Central Repository
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'adopt'
```

Comment:
Why two different distros in the same workflow?

Reply:
Cause copy and paste? ;-)

Reply:
oh, it's even in the same file! I thought you meant the release has a different distro than the build. I'll fix this!

Reply:
fixed.